misc/config_tools/board_inspector/extractors/65-usb.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2022 Intel Corporation. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+import os, re
+
+from extractors.helpers import add_child, get_node
+
+USB_DEVICES_PATH = "/sys/bus/usb/devices"
+USB_DEVICES_REGEX = r"^\d-\d$"   # only devices connecting to root hub
+
+def extract(args, board_etree):
+    usb_devices_node = get_node(board_etree, f"//usb_devices")
```

Comment:
Physical devices that can be enumerated shall be put under the "devices" node rather than creating a new one.

For USB devices, get the BDF of the controller via the real path and append the devices under the corresponding node that has been created by the PCI extractor.

Reply:
Put it under the PCI device or under the exact acpi port device ?

Reply:
Is there any standard identifier of such ACPI port devices, or any standard approach to get such identifiers? If it is model specific, you can put those devices under the PCI device directly. Otherwise under the ACPI port device is my preference, but the former is still acceptable considering schedule constraints.